media-gfx/openvdb/openvdb-3.2.0.ebuild

```diff
@@ -0,0 +1,172 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $id$
+
+EAPI="6"
+PYTHON_COMPAT=( python{2_7,3_4,3_5} )
+
+inherit eutils multilib python-r1
+
+DESCRIPTION="Libs for the efficient manipulation of volumetric data"
+HOMEPAGE="http://www.openvdb.org"
+
+SRC_URI="https://github.com/dreamworksanimation/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MPL-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+openvdb-compression doc pdfdoc python X"
+
+REQUIRED_USE="${PYTHON_REQUIRED_USE}
+	pdfdoc? ( doc )"
+
+RDEPEND="${PYTHON_DEPS}"
+
+DEPEND="${RDEPEND}
+	sys-libs/zlib
+	>=dev-libs/boost-1.62:=[${PYTHON_USEDEP}]
+	media-libs/openexr
+	dev-cpp/tbb
+	dev-util/cppunit
+	doc? (
+		app-doc/doxygen
+		python? ( dev-python/pdoc[${PYTHON_USEDEP}] )
+	)
+	pdfdoc? (
+		app-doc/doxygen[dot,latex]
+		app-text/ghostscript-gpl
+	)
+	X? ( media-libs/glfw )
+	dev-libs/jemalloc
+	python? ( dev-python/numpy[${PYTHON_USEDEP}] )
+	openvdb-compression? ( >=dev-libs/c-blosc-1.5.0 )
+	dev-libs/log4cplus"
+
+S="${WORKDIR}/${P}/${PN}"
+
+PATCHES=(
+	"${FILESDIR}"/${P}-numpy-fix.patch
+	"${FILESDIR}"/${P}-makefile-fixes.patch
+)
+
+src_prepare() {
+	default
+
+	sed	-e "s|--html -o|--html --html-dir|" \
+		-e "s|vdb_render vdb_test|vdb_render vdb_view vdb_test|" \
+		-i Makefile || die "sed failed"
+
+	sed	-e "s|= png|= svg|" -i doxygen-config || die "sed doxygen failed"
```

Comment:
Can you turn both of these seds into a patch please? Then you can also get rid of `src_prepare` altogether